src/BTrees/Interfaces.py

```diff
@@ -52,9 +45,9 @@ def has_key(key):
         """
 
     def keys(min=None, max=None, excludemin=False, excludemax=False):
-        """Return an :class:`~BTrees.Interfaces.IReadSequence` containing the keys in the collection.
+        """Return an :class:`~BTrees.Interfaces.IMinimalSequence` containing the keys in the collection.
```

Comment:
Maybe this should point to `zope.interface.common.sequence.IMinimalSequence` as the actual interface. (Same for the other changes previously referencing `IReadSequence`.)

Reply:
OK.